library/tiqr/Tiqr/Exception/ReadWriteException.php

```diff
@@ -18,4 +18,9 @@
 
 class ReadWriteException extends RuntimeException
 {
+    public static function fromOriginalException(Exception $e): ReadWriteException
+    {
+        // $code must be int, otherwise this throws with Error("Wrong parameters for ReadWriteException")
```

Comment:
For me personally, adding this comment is not super relevant. It's not in the way either. 

My rule of thumb is to make the code speak for itself (self documenting). If that is not possible. Or complexity is to high, a comment may be used to explain to the future developer what's happening.

Reply:
I'll treat it as a bug in my comment.
It's there too explain that the cast to int is not spurious but that it is there to because PDOException returns code of type string.